bitnami/grafana/templates/deployment.yaml

```
@@ -79,7 +79,12 @@ spec:
           image: {{ include "volumePermissions.image" . }}
           imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
           command:
-            - chown -R {{ .Values.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }} /bitnami/grafana
+            - /bin/bash
+            - '-ec'
+          args:
+            - |
```

Actually the `-ec` are part of the args:
```suggestion
          args:
            - -ec
            - |
```

---

Yes good point! I suppose this could also be changed in the other Bitnami charts.